src/redisearch_rs/trie_bencher/build.rs

```diff
@@ -18,7 +18,6 @@ fn main() {
     let lib_dir = {
         let target_arch = env::var("CARGO_CFG_TARGET_ARCH").unwrap();
         let target_arch = match target_arch.as_str() {
-            "aarch64" => "arm64v8",
```

Comment:
Relates to the build changes, right? not to the u8 change

Reply:
Indeed. Without this, the microbenchmarks won't work in CI (nor locally, if using `build.sh` to do the build). Same for the change in the workflow.